lib/main.dart

```diff
@@ -79,14 +67,14 @@ class RandomWordsState extends State<RandomWords>{
     final wordPair = WordPair.random();
     return Scaffold(
         appBar: AppBar(
+          leading: Icon(Icons.account_circle),
```

Comment:
plz write comments with changes u have done

Reply:
Understand, got it!